openstack_controller/tests/test_end_to_end.py

```diff
@@ -7120,3 +7121,14 @@ def test_scenario(make_request, aggregator):
 
         # Assert coverage for this check on this instance
         aggregator.assert_all_metrics_covered()
+
+
+@pytest.mark.integration
```

Comment:
Not sure if I want to set up a terraform environment by mistake while running `ddev test`. What do you think, would that be enough to keep as `env start` and `env stop` ?

Reply:
Turned it into e2e.